CMakeLists.txt

```diff
@@ -167,39 +165,65 @@ add_executable(${TEST_NAME} ${Src})
 if(${hsa-runtime64_FOUND})
    target_link_libraries(${TEST_NAME} PRIVATE hsa-runtime64::hsa-runtime64)
 else()
-   target_link_libraries(${TEST_NAME} PRIVATE ${ROCR_LIB} ${ROCT_LIB} )
+   target_link_libraries(${TEST_NAME} PRIVATE ${ROCR_LIB} ${ROCT_LIB})
 endif()
 target_link_libraries(${TEST_NAME} PRIVATE c stdc++ dl pthread rt)
 
 # Update linker flags to include RPATH
 # Add --enable-new-dtags to generate DT_RUNPATH
 if( DEFINED ENV{ROCM_RPATH})
-   set ( CMAKE_EXE_LINKER_FLAGS "-Wl,--enable-new-dtags -Wl,--rpath,$ENV{ROCM_RPATH}" )
+   set ( CMAKE_EXE_LINKER_FLAGS "-Wl,--enable-new-dtags -Wl,--rpath,$ENV{ROCM_RPATH}")
 endif()
 
 # Add install directives for rocm_bandwidth_test
 install(TARGETS ${TEST_NAME} RUNTIME DESTINATION bin)
 
 # Add packaging directives for rocm_bandwidth_test
 set(CPACK_PACKAGE_NAME ${PROJECT_NAME})
-set(CPACK_PACKAGE_VENDOR "AMD")
+set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
 set(CPACK_PACKAGE_VERSION_MAJOR ${BUILD_VERSION_MAJOR})
 set(CPACK_PACKAGE_VERSION_MINOR ${BUILD_VERSION_MINOR})
 set(CPACK_PACKAGE_VERSION_PATCH ${BUILD_VERSION_PATCH})
-set(CPACK_PACKAGE_CONTACT "Advanced Micro Devices Inc.")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Test to measure PciE bandwidth on ROCm platforms")
+set(CPACK_PACKAGE_CONTACT "TODO <Add a valid email id>")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Diagnostic utility tool to measure PCIe bandwidth on ROCm platforms")
+
+#Make proper version for appending
```

Comment:
Please add a space separator, here and other places

Reply:
Ack!!
Posted the new commit